README.md

```diff
@@ -1,5 +1,5 @@
 
-### ![#f03c15](https://placehold.it/15/f03c15/000000?text=+) Community Announcements ![#f03c15](https://placehold.it/15/f03c15/000000?text=+)
```

Comment:
Images around the 'Community Announcements' title, introduced with https://github.com/typelevel/cats/pull/2683, and served by https://placehold.it/ are unfortunately broken:
![image](https://user-images.githubusercontent.com/52038/121752959-2fadde00-cb09-11eb-98f3-2c4456edbcac.png)